src/sentry/monitors/logic/mark_failed.py

```diff
@@ -55,7 +56,7 @@ def mark_failed(
 
     # When the failed check-in is a synthetic missed check-in we do not move
     # the `last_checkin` timestamp forward.
-    if failed_checkin.status == CheckInStatus.MISSED:
+    if is_miss:
```

Comment:
i'm kind of confused why we need this boolean instead of relying on the checkin status

Reply:
Because the check-in status will be UNKNOWN in the future